backend/Squirrel.SqlService/Squirrel.SqlService.BLL/Models/DTO/DbStructureDto.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Squirrel.SqlService.BLL.Models.DTO.Function;
+using Squirrel.SqlService.BLL.Models.DTO.Procedure;
+using Squirrel.SqlService.BLL.Models.DTO.View;
+
+namespace Squirrel.SqlService.BLL.Models.DTO;
+
+public class DbStructureDto
+{
+    public ICollection<TableStructureDto>? TableStructures { get; set; }
+    public ICollection<TableConstraintsDto>? Constraints { get; set; }
+    public FunctionDetailsDto? FunctionDetails { get; set; }
+    public ProcedureDetailsDto? ProcedureDetails { get; set; }
+    public ViewDetailsDto? ViewDetails { get; set; }
```

Comment:
it seems we can have it consolidated - to have `ICollection` for all or a list inside models for all because we should have a list of tables, a list of SP, and so on, doesn't it?